src/qminer/qminer_core.cpp

```diff
@@ -5596,46 +5605,70 @@ void TIndex::DoQueryPos(const int& KeyId, const TUInt64V& WordIdV,
     if (WordIdV.Empty()) { return; }
     // get records for the first word from the index and
     // store it into the running result candidate vector
-    TVec<TQmGixItemPos> ItemV;
-    GixPos->GetItemV(TQmGixKey(KeyId, WordIdV[0]), ItemV);
-    ItemV.Sort();
+    TVec<TQmGixItemPos> CurrentItemV;
+    GixPos->GetItemV(TQmGixKey(KeyId, WordIdV[0]), CurrentItemV);
+    Assert(CurrentItemV.IsSorted());
+    //ItemV.Sort();
```

Comment:
Why does `IsSorted` now hold? Was there not some issue with this assumption last time?